ontology/yaml/resources/ELECTRICAL/entity_types/ABSTRACT.yaml

```diff
@@ -183,3 +183,13 @@ SRC2:
   - source2_phase3_phase1_over_voltage_status
   - source2_over_voltage_status
   - source2_lost_power_alarm
+
+BBM:
+  guid: "6ea6a214-a12f-4f8c-b97b-512538566611"
+  description: "For UPS with Load Bypass and Battery status."
+  is_abstract: true
+  uses:
+  - bypass_status
```

Comment:
`bypass_status` is already included on the `UPS` type, so please remove from this type and make this type more general to just model battery status monitoring.

Reply:
I see, will do.